vmdb/app/helpers/container_node_helper/textual_summary.rb

```diff
@@ -15,4 +15,12 @@ def textual_group_properties
   def textual_name
     {:label => "Name", :value => @record.name}
   end
+
+  def textual_creation_timestamp
+    {:label => "Creation Timestamp", :value => @record.creation_timestamp}
```

Comment:
@erinboyd use format_timezone(@record.creation_timestamp) that should format time for you.